apps/service_aggregate/lib/aggregate/event/handler.ex

```diff
@@ -0,0 +1,33 @@
+defmodule Aggregate.Event.Handler do
+  @moduledoc """
+  Callbacks for handling events from `Brook`.
+  """
+  use Brook.Event.Handler
+  require Logger
+
+  import Events, only: [extract_start: 0, aggregate_update: 0]
+  import Definition, only: [identifier: 1]
+
+  def handle_event(%Brook.Event{
+        type: extract_start(),
+        data: %Extract{destination: %Kafka.Topic{}} = extract
+      }) do
+    Logger.debug(fn ->
+      "#{__MODULE__}: Received event #{extract_start()}: #{inspect(extract)}"
+    end)
+
+    Aggregate.Feed.Supervisor.start_child(extract)
+
+    identifier(extract)
+    |> Aggregate.ViewState.Extractions.persist(extract)
+  end
```

Comment:
@bbalser do we want to comment this one out or delete it entirely before merging this in? This seems like it would be the only thing that could cause issues (even just resource contention) as it would trigger aggregate and profile to do the same thing on _any_ new dataset ingestion.

Reply:
maybe not; since it's not actually doing the profile/aggregate, just storing an extract in state.

Reply:
I think it is okay to leave